app/app.config.ts

```diff
@@ -8,37 +8,52 @@ import { version } from './package.json'
 module.exports = {
   expo: {
     name: 'STA',
-    displayName: 'STA Volunteers',
-    slug: 'sta',
+    slug: 'volunteer-app',
     version,
     orientation: 'portrait',
     icon: './assets/sta-icon.png',
-    userInterfaceStyle: 'light',
+    userInterfaceStyle: 'automatic',
```

Comment:
This is what fixes the issue with the app not automatically picking up the device's dark/light mode (stumbled across this along the way!)